docs/content/direct/core-chart.md

```diff
@@ -115,11 +119,18 @@ In particular:
 - `kubeflex-operator.install` accepts a boolean value to enable/disable the installation of KubeFlex into the cluster by the chart
 - `kubeflex-operator.isOpenShift` must be set to true by the user when installing the chart into a OpenShift cluster
 
-By default, the chart will install the KubeFlex and its PostgreSQL dependency.
+By default, the chart will install KubeFlex and its PostgreSQL dependency. Post Create Hooks (PCHs) needed for creating ITSes and WDSes control planes are automatically managed using intelligent detection:
 
-The second section allows a user of the chart to determine if Post Create Hooks (PCHes) needed for creating ITSes and WDSes control planes should be installed by the chart. By default `InstallPCHs` is set to `true` to enable the installation of the PCHes, however one may want to set this value to `false` when installing multiple copies of the chart to avoid conflicts. A single copy of the PCHes is required and allowed per cluster.
+**Automatic PCH Management:**
+- **First install**: PCHs are installed automatically because they don't exist in the cluster
+- **Additional deployments**: PCHs are skipped automatically because they already exist
+- **No manual configuration needed** - the chart detects what's appropriate
 
-The third section of the `values.yaml` file allows one to create a list of Inventory and Transport Spaces (ITSes). By default, this list is empty and no ITS will be created by the chart. A list of ITSes can be specified using the following format:
+**Advanced PCH Control:**
+- Set `InstallPCHs=true` to force reinstall PCHs (useful for chart upgrades)

```

Comment:
This is not "advanced". It is non-optional consideration when doing `helm upgrade` of the first instance, and forbidden when `helm upgrade`ing any other instance.